packages/app-mobile/utils/shim-init-react/index.ts

```diff
@@ -31,6 +31,10 @@ export default function shimInit() {
 		return temp;
 	};
 
+	shim.announceForAccessibility = (message) => {
+		AccessibilityInfo.announceForAccessibility(message);
+	};
```

Comment:
Currently, this implements `shim.announceForAccessibility` on Android and iOS, but not the web version of the mobile app.